indico/modules/auth/controllers.py

```diff
@@ -593,3 +593,14 @@ def _reset_password(self, identity):
         form.username.data = identity.identifier
         return WPAuth.render_template('reset_password.html', form=form, identity=identity,
                                       widget_attrs={'username': {'disabled': True}})
+
+
+class RHSessionsInvalidate(RHUserBase):
```

Comment:
I'd literally flipped 3 times between the RHSessionsInvalidate and RHInvalidateSessions before pushing. I think the only reason I went with RHSessionsInvalidate was because the I thought I should match the URL which seemed right as /sessions/invalidate.